### ISSUE TITLE
Receive error in console when going to order summary console

### ISSUE THREAD

**Issue by zenweasel** [reactioncommerce/reaction#2337]
### Expected behavior
No errors in console

### Actual Behavior
Receive this error
```
modules.js?hash=c87d7d4…:10098 Warning: React.createElement: type is invalid -- expected a string (for built-in components) or a class/function (for composite components) but got: null. Check the render method of `Container(OrderSummaryContainer)`.
```

### Steps to Reproduce the Behavior
1. Place an order
1. Login an admin
1. Go to order panel
1. Observe error

### Versions
```
Node: 7.10.0
NPM: 4.2.0
Meteor Node: 4.8.2
Meteor NPM: 4.5.0
Reaction CLI: 0.9.1
Reaction: 1.2.0
Reaction branch: development
Docker: 17.04.0-ce
````

**Comment by aaronjudd** [comment]
Confirmed, I've seen this as well. @kieckhafer you may want to take this as part of orders milestone?

**Comment by kieckhafer** [comment]
Ahh, so this isnt' just in my branch... I thought it was something I was doing, guess it was from my most recent pull from dev... 

I have the solution, one line of code, i will make a PR outside of my branch to fix so it gets pulled in quickly.

**Comment by zenweasel** [comment]
Closed via #2340
